nvme.pyx

```diff
@@ -53,3 +121,156 @@ def get_nsid(path):
         return nsid.cdev
     finally:
         os.close(fd)
+
+
+cdef class NvmeDevice(object):
+
+    cdef const char *dev
+    cdef int fd
+
+    def __cinit__(self, path):
+
+        self.dev = path
+        self.fd = open(self.dev, O_RDONLY)
```

Comment:
Wonder if open/close could block for some? If so we would need to release GIL

Reply:
The entire purpose for me brining in fcntl standard open and close functions was to release the gil when opening the device. I just simply forgot to add those parameters. Great catch, I'll update.